OrbitGl/SchedulerTrack.h

```diff
@@ -12,6 +12,9 @@ class SchedulerTrack : public ThreadTrack {
   void UpdatePrimitives(uint64_t min_tick, uint64_t max_tick) override;
   Type GetType() const override { return kSchedulerTrack; }
   float GetHeight() const override;
+
+  protected:
+  float GetYFromDepth(float track_y, uint32_t depth, bool collapsed) override;
```

Comment:
While this is not a public API, I would prefer not to use booleans in interfaces as much as possible. Using an enum will make it more obvious what is happening at callsites (and avoids the additional /* collapsed */ comment) and also makes it easier to expand in the future (for example, a track could have states COLLAPSED, SUMMARY, and FULL_DETAIL).